server/channels/store/sqlstore/user_store.go

```diff
@@ -1358,16 +1358,23 @@ func (us SqlUserStore) AnalyticsActiveCount(timePeriod int64, options model.User
 	time := model.GetMillis() - timePeriod
 	query := us.getQueryBuilder().Select("COUNT(*)").From("Status AS s").Where("LastActivityAt > ?", time)
 
-	if !options.IncludeBotAccounts {
-		query = query.LeftJoin("Bots ON s.UserId = Bots.UserId").Where("Bots.UserId IS NULL")
-	}
-
-	if !options.IncludeDeleted {
-		query = query.LeftJoin("Users ON s.UserId = Users.Id").Where("Users.DeleteAt = 0")
+	if us.DriverName() == model.DatabaseDriverPostgres {
+		if !options.IncludeBotAccounts {
+			query = query.LeftJoin("Bots ON s.UserId = Bots.UserId").Where("Bots.UserId IS NULL")
+		}
+		if !options.IncludeDeleted {
+			query = query.LeftJoin("Users ON s.UserId = Users.Id").Where("Users.DeleteAt = 0")
```

Comment:
I don't think we need to do this for the users table. This seems like a slightly different query. Was this also resulting in an antijoin?

Reply:
Hmm, thought it would result in the same way and didn't check it TBH. Does `IS NULL` behave differently from `DeleteAt = 0`?

Reply:
Yes it would. `IS NULL` would mean the rows that didn't match resulting in an antijoin. `DeleteAt = 0` would be a normal join with a filter. That's what I am assuming without actually running those queries through an EXPLAIN.

Reply:
Right, did run an explain analyze and there is no antijoin. Reverted for users table 👍